polyfills/UserTiming/config.json

```diff
@@ -0,0 +1,18 @@
+{
+	"browsers": {
+		"android": "<=4",
+		"chrome": "<=24",
+		"firefox": "<=37",
+		"firefox_mob": "*",
+		"ie": "6 - 9",
+		"ie_mob": "<=9",
+		"ios_chr": "*",
+		"ios_saf": "*",
+		"op_mob": "*",
+		"opera": "<=12",
+		"safari": "*"
+	},
+	"spec": "http://www.w3.org/TR/user-timing/",
+	"license": "MIT",
+	"repo": "https://github.com/nicjansma/usertiming.js"
```

Comment:
Can you add an appropriate `docs` link, eg https://developer.mozilla.org/en-US/docs/Web/API/User_Timing_API?